paasta_tools/cli/schemas/kubernetes_schema.json

```diff
@@ -628,11 +628,11 @@
                     "type": "array"
                 },
                 "iam_role": {
-                    "type": "string"
+                    "type": "string",
+                    "pattern": "^arn:aws:iam::[0-9]+:role/[a-zA-Z0-9+=,.@_-]+$"
```

Comment:
should we link to https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_iam-quotas.html#reference_iam-quotas-names above this (here and in the tron schema)?

i can't really see AWS changing this format at all, but might be worth it just in case :p